### PR TITLE
Update match engine 

### PR DESCRIPTION
**Pull request by alastairong**
Adds:
- auto-update
- Post HDK version of holochain
- Latest HHA and servicelogger

### CODE REVIEW

File: profiles/logical/holo/match-server/default.nix [hunk 1]
```diff
@@ -10,24 +10,28 @@ let
...
 
-  services.holochain = {
+  services.holo-envoy.enable = lib.mkDefault true; # Enabling because Holochain systemd service depends on it. We're not using it or lair-shim
+
+  services.holochain = lib.mkDefault {
     enable = true;
     working-directory = holochainWorkingDir;
     config = {
-      environment_path = "${holochainWorkingDir}/databases";
+      environment_path = "${holochainWorkingDir}/databases_lmdb4";
```
Comment (peeech) [inline profiles/logical/holo/match-server/default.nix:34]: How are we gonna keep those in sync across profiles?

File: profiles/logical/holo/match-server/default.nix [hunk 2]
```diff
@@ -59,18 +76,24 @@ in
...
+       {
+         app_id = "core-app";
+         bundle_url = "https://holo-host.github.io/holo-hosting-app-rsm/releases/downloads/0_1_0_alpha17/core-app.0_1_0_alpha17.happ";
+       }
+       {
+         app_id = "servicelogger";
+         bundle_url = "https://holo-host.github.io/servicelogger-rsm/releases/downloads/v0.1.0-alpha7/servicelogger.0_1_0-alpha7.happ";
+       }
+      ];
+    };
```
Comment (peeech) [inline profiles/logical/holo/match-server/default.nix:89]: How are we gonna keep those in sync across profiles?

File: profiles/logical/holo/match-server/default.nix [hunk 3]
```diff
@@ -39,17 +43,30 @@ in
...
           proxy_config = {
             type = "remote_proxy_client";
-            proxy_url = "kitsune-proxy://nFCWLsuRC0X31UMv8cJxioL-lBRFQ74UQAsb8qL4XyM/kitsune-quic/h/proxy.holochain.org/p/5775/--";
+            proxy_url = kitsuneAddress;
           };
         }];
+        tuning_params = {
+          gossip_loop_iteration_delay_ms = 1000; # Default was 10
+          default_notify_remote_agent_count = 5;
+          default_notify_timeout_ms = 1000;
```
Comment (zippy) [inline profiles/logical/holo/match-server/default.nix:61]: I'm worried about these `tuning_params`  We should remove all of the ones that don't have a `# Default was...` commint. We also need to delete those non default overrides in the base profile.

**Review by zippy — CHANGES_REQUESTED** [review]

File: profiles/logical/holo/match-server/default.nix (hunk 1 above)
Comment (alastairong) [reply profiles/logical/holo/match-server/default.nix:34]: Does the database name need to be kept in sync across profiles?

File: profiles/logical/holo/match-server/default.nix (hunk 2 above)
Comment (alastairong) [reply profiles/logical/holo/match-server/default.nix:89]: Yea maybe we should extract it so we can unify across profiles.

File: profiles/logical/holo/match-server/default.nix (hunk 3 above)
Comment (alastairong) [reply profiles/logical/holo/match-server/default.nix]: @zippy See [22af9ae](https://github.com/Holo-Host/holo-nixpkgs/pull/835/commits/22af9aec84b07371e3a78a3b3dae16a7828caf7c)

Is it ok to have missing params? Are these overrides with defaults?

File: profiles/logical/holo/match-server/default.nix [hunk 4]
```diff
@@ -10,24 +10,28 @@ let
   holochainWorkingDir = "/var/lib/holochain-rsm";
 
   configureHolochainWorkingDir = "/var/lib/configure-holochain";
+
+  kitsuneAddress = "kitsune-proxy://f3gH2VMkJ4qvZJOXx0ccL_Zo5n-s_CnBjSzAsEHHDCA/kitsune-quic/h/45.55.107.33/p/5788/--";
```
[the remark this answers is not in the document]
Comment (alastairong) [reply profiles/logical/holo/match-server/default.nix:14]: We should extract it elsewhere. But that should be it's own PR I think

**Review by zippy — APPROVED** [review]

**Pull request merged**